composer.json

```diff
@@ -9,7 +9,8 @@
     "laravel/tinker": "^1.0",
     "aws/aws-sdk-php-laravel": "^3.1",
     "barryvdh/laravel-debugbar": "^2.3",
-    "doctrine/dbal": "^2.5",
+    "doctrine/dbal": "~2.5.13",
+    "doctrine/common": "~2.6.0",
```

Comment:
@DFurnes Removed `doctrine/common`, ran `composer update` and pushed up the new lock file, getting this in wercker again:
```
 Problem 1
    - Installation request for doctrine/annotations v1.5.0 -> satisfiable by doctrine/annotations[v1.5.0].
    - doctrine/annotations v1.5.0 requires php ^7.1 -> your PHP version (7.0.21) does not satisfy that requirement.
  Problem 2
    - Installation request for doctrine/cache v1.7.0 -> satisfiable by doctrine/cache[v1.7.0].
    - doctrine/cache v1.7.0 requires php ~7.1 -> your PHP version (7.0.21) does not satisfy that requirement.
  Problem 3
    - Installation request for doctrine/collections v1.5.0 -> satisfiable by doctrine/collections[v1.5.0].
    - doctrine/collections v1.5.0 requires php ^7.1 -> your PHP version (7.0.21) does not satisfy that requirement.
  Problem 4
    - Installation request for doctrine/instantiator 1.1.0 -> satisfiable by doctrine/instantiator[1.1.0].
    - doctrine/instantiator 1.1.0 requires php ^7.1 -> your PHP version (7.0.21) does not satisfy that requirement.
  Problem 5
    - doctrine/annotations v1.5.0 requires php ^7.1 -> your PHP version (7.0.21) does not satisfy that requirement.
    - doctrine/common v2.7.3 requires doctrine/annotations 1.* -> satisfiable by doctrine/annotations[v1.5.0].
    - Installation request for doctrine/common v2.7.3 -> satisfiable by doctrine/common[v2.7.3].
```